apis/python/src/tiledbsoma/io/ingest.py

```diff
@@ -1722,9 +1726,10 @@ def add_X_layer(
     add_matrix_to_collection(
```

Comment:
This is public API and we can't change it without a deprecation cycle, or, if it is necessary to break API, to offer our users fair warning and an example of how they will need to change their notebooks/scripts/etc to accommodate the API breakage this PR would cause them.

Reply:
Correction: you're not modifying public API, you're calling it kwargishly in callsites that used to call positionally. This is fine. Nonetheless, please make the other changes requested.

Reply:
I do want a PR (at least one, or more) which touches _only_ the cosmetics and not unit-test files.

As we've discussed, our unit-test coverage is our bulwark line of defense against any breakage of ingest. There are myriad cases of H5AD/AnnData input that have raised bug reports over the years, and unit-test coverage is our _sole_ way of guaranteeing we aren't regressing.